scenes/game/local_game.gd

```diff
@@ -2285,6 +2519,10 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			performing_player.strike_stat_boosts.ignore_guard = true
 		"ignore_push_and_pull":
 			performing_player.strike_stat_boosts.ignore_push_and_pull = true
+		"ignore_push_and_pull_boost":
+			performing_player.ignore_push_and_pull = true
```

Comment:
maybe we should call this one "ignore_push_and_pull_boost_bonus" just to be super clear and not confuse with the other one
Or maybe passive_bonus? passive_boost?